ionic/src/app/api/api.service.ts

```diff
@@ -36,7 +37,7 @@ export class ApiService {
     // return await this.http.get<Wifi[]>(this.base_url + "wifi").toPromise();
     try {
       const data = await this.http.get(`${environment["apiBase"]}deviceSimulator/scanWifi`).toPromise();
```

Comment:
const data is not required anywhere. simply do 
return await this.http.get(`${environment["apiBase"]}deviceSimulator/scanWifi`).toPromise();